content/docs/introducing-jsx.md

```diff
@@ -18,6 +18,16 @@ It is called JSX, and it is a syntax extension to JavaScript. We recommend using
 
 JSX produces React "elements". We will explore rendering them to the DOM in the [next section](/docs/rendering-elements.html). Below, you can find the basics of JSX necessary to get you started.
 
+### Why JSX?
+
+React embraces the fact that rendering logic is inherently coupled with other UI logic: how events are handled, how the state changes over time, and how the data is prepared for display.
+
+Instead of artificially separating *technologies* by putting markup and logic in separate files, React [separates *concerns*](https://en.wikipedia.org/wiki/Separation_of_concerns) with loosely coupled units called "components" that contain both. We will come back to components in a [further section](/docs/components-and-props.html), but if you're not yet comfortable putting markup in JS, [this talk](https://www.youtube.com/watch?v=x7cQ3mrcKaY) might convince you otherwise.
```

Comment:
How about

> Instead of artificially separating *technologies* by putting markup and logic in separate files, React [separates *concerns*](https://en.wikipedia.org/wiki/Separation_of_concerns) ~~with~~ **into** loosely coupled units called "components" that contain both. We will come back to components in a [further section](/docs/components-and-props.html), but if you're not yet comfortable putting markup in JS, [this talk](https://www.youtube.com/watch?v=x7cQ3mrcKaY) might convince you otherwise.

Something about saying ```this talk might convince you otherwise``` sounds a little off-putting to me. Maybe, just ```if you're not yet comfortable..., check out this talk```?

Reply:
I dunno. That's how I replied to folks on Twitter in the past and it's been effective :-) But I'm open to other wordings if you feel strongly—although I'm not the best person to review since I'm not a native speaker.

Reply:
What I think this means is that React is redefining "concerns" to mean "pieces of app logic" _instead of_ "layout, state, and mutation". How about:

> Instead of artificially separating _technologies_ by putting markup and logic in separate files, React separates _business concerns_ into loosely coupled units called "components" that contain both. We will come back to components in a further section, but if you're not yet comfortable putting markup in JS, this talk might convince you otherwise.

Reply:
I wouldn't put "business concerns" there. Sounds very enterprisey. "Separation of concerns" is a well-known CS principle, even though people don't always agree what it means.

Is is okay for this paragraph to sting a little bit and be slightly controversial. Those who won't be persuaded won't be anyway. But I think this argument is strong enough to sway the people who are on the fence.

Reply:
Doesn't have to be "business concerns"; I agree that it has an enterprisey connotation. But "separate concerns **into** components" is not grammatical; I'd stick with `with` or `using`.

Reply:
Oh I see. Yeah let's leave it :-)